editoast/src/views/timetable/stdcm.rs

```diff
@@ -411,6 +426,47 @@ fn get_earliest_departure_time(
     )
 }
 
+// Return the latest time at which the train may arrive, margins included.
```

Comment:
```suggestion
/// Return the latest time at which the train may arrive, margins included.
```

`///` are doc comments, `//` and `/* */` are regular comments